README

```diff
@@ -39,7 +39,10 @@
     libgcrypt    (ftp://ftp.gnupg.org/gcrypt/libgcrypt/)
     libksba      (ftp://ftp.gnupg.org/gcrypt/libksba/)
     libassuan    (ftp://ftp.gnupg.org/gcrypt/libassuan/)
-
+  
+  Downgrade maybe required. For example, on Ubuntu 16.04 latest version of libgcrypt will not work. 
+  In that case we need downgrade the library.
```

Comment:
Grammatical suggestion:

Downgrade maybe required. For example, on Ubuntu 16.04, the latest version of libgcrypt will not work. In this case you will need to downgrade the library.

I also have no other context but I'd recommend providing more context as to why it doesn't work, like a blog post or something like that.

Reply:
Thank you so much.